templates/unit.erb

```diff
@@ -0,0 +1,38 @@
+[Unit]
+Description=Apache Kafka server (<%= (@service_name.split(/-/)[1] and @service_name.split(/-/)[1].capitalize) or 'broker' -%>)
+Documentation=http://kafka.apache.org/documentation.html
+Wants=zookeeper.service
+After=zookeeper.service
+Requires=zookeeper.service
```

Comment:
please stop reminding me :D

maybe we should define a policy/guideline somewhere which mentions a way to handle service files. So we can make it consistent across our modules (in a far far future)